indico/modules/events/contributions/controllers.py

```diff
@@ -95,8 +79,40 @@ class RHContributions(RHManageContributionsBase):
     """Display contributions management page"""
 
     def _process(self):
+        contrib_report_config = reporter.get_report_config(self.event_new.id)
+        if 'config' in request.args:
```

Comment:
Why not return two values from `get_report_config`, one to indicate whether a stored config has been loaded. Then you can use this to check whether to redirect. Right now you need to check `request.args['config']` in two pretty different places.